packages/account/src/Sections/Security/Passwords/passwords-static.jsx

```diff
@@ -0,0 +1,74 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Localize, localize } from '@deriv/translations';
+import { Icon, Popover, Text } from '@deriv/components';
+import { connect } from 'Stores/connect';
+import DerivComLogo from '../../../Assets/ic-brand-deriv-red.svg';
+import DerivGoLight from '../../../Assets/ic-brand-deriv-go-light.svg';
+import DerivGoDark from '../../../Assets/ic-brand-deriv-go-dark.svg';
```

Comment:
```suggestion
import DerivGoDark from 'Assets/ic-brand-deriv-go-dark.svg';
```

Reply:
Not applicable as we are missing alias in Account component